pymc/model_graph.py

```diff
@@ -219,13 +242,15 @@ def make_graph(self, formatting: str = "plain"):
                 for var_name in var_names:
                     self._make_node(var_name, graph, formatting=formatting)
 
-        for key, values in self.make_compute_graph().items():
-            for value in values:
-                graph.edge(value.replace(":", "&"), key.replace(":", "&"))
+        for child, parents in self.make_compute_graph(selected_vars=selected_vars).items():
+            # parents is a set of rv names that preceed child rv nodes
+            for parent in parents:
+                graph.edge(parent.replace(":", "&"), child.replace(":", "&"))
+
         return graph
 
 
-def model_to_graphviz(model=None, *, formatting: str = "plain"):
+def model_to_graphviz(model=None, *, selected_vars=None, formatting: str = "plain"):
```

Comment:
Let's align it with ArviZ: `var_names`

The ArviZ documentation describes it as:
```
var_names: list[str], optional
    List of variables to plot (defaults to None, which results in all variables plotted).
    Prefix the variables by ~ when you want to exclude them from the plot.
```

Reply:
I don't see a problem with using it as the argument.
You could even rename the `self.var_names` to `self._all_var_names` or so. They should have been private to begin with..

Reply:
Done!